requests_auth/oauth2_tokens.py

```diff
@@ -92,11 +94,12 @@ def get_token(self, key: str, on_missing_token=None, *on_missing_token_args) ->
         :raise AuthenticationFailed: in case token cannot be retrieved.
         """
         logger.debug(f'Retrieving token with "{key}" key.')
+        refresh_token = None
         with self.forbid_concurrent_cache_access:
             self._load_tokens()
             if key in self.tokens:
-                bearer, expiry = self.tokens[key]
-                if is_expired(expiry):
+                bearer, expiry, refresh_token = self.tokens[key]
```

Comment:
What about the case were user is using a static cache for tokens? This will crash because cache will not yet contain a refresh token?

Reply:
I added an extra check on the length of the returned tuple. Now it should work both for cached tokens with or without a refresh token.

Reply:
Also added a test for this case. Code coverage by tests is back up to 100%.